### DISCUSSION TITLE
Do you need volunteers? What skillset?

### DISCUSSION THREAD

**Discussion by yesicanhearyouclemfandango** [openstates/issues#424] [General] [answered]
I am not technically inclined, but I know a lot of people who are, and I'd love to send some your way, if you need them. In particular, I have some close connections to the local Python community.

I want this project to thrive! Who do you need? What do you need?

Kevin Farmer, NC

**Comment by jamesturk** [comment, marked as answer]
Hi Kevin,

Thanks for reaching out!

We generally do need volunteers, in fact we're going to be launching a couple of pushes for improved data soon that'd really benefit from two types of volunteer:

- General data entry volunteers are helpful for tracking down data that can't be scraped.  My colleague @NewAgeAirbender has been building a UI for augmenting our people data, and I'm excited that we'll soon be able to ask people to track down social media accounts/etc. for legislators.  If you have people that would like to track down manual data that are a bit less technical but comfortable with some googling/using a spreadsheet or similar, we'll have plenty for them to do very soon.
- We're also about to start to push forward on collecting committee data again, that means a bunch of new scrapers and we can use Python developers that would like to scrape some new data.  

Both of these are going to be a focus through June/July I'd imagine, I'll be sharing more specifics soon on the Open States blog and in the public Slack.  

Thanks for offering to help find volunteers,

James

**Reply by yesicanhearyouclemfandango** [reply]
Hi James, 

I can find you plenty of people who can do the former. I will certainly do my best. I forwarded this link to a friend who runs the Triangle Python group.

Added the blog to Feedly and followed on Twitter. Standing by!

-K
